app-profile-jee-html5/src/main/webapp/app.js

```diff
@@ -43,7 +43,7 @@ function show(id) {
 keycloak.onAuthLogout = welcome;
 
 window.onload = function () {
-    keycloak.init({ onLoad: 'check-sso', checkLoginIframeInterval: 1 }).success(function () {
+    keycloak.init().success(function (authenticated) {
```

Comment:
Don't remove check-sso as it was expected to be there. Not sure why checkLoginIframeInterval=1 was here before, perhaps a work around to get the tests to work?

Reply:
Reverted this change.